app/models/message_search.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+# MessageSearch model
+class MessageSearch < Common::Base
+  # include RedisCaching
```

Comment:
I would go ahead and delete these commented lines.

Reply:
Deleted!